sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_statistics_v2/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+friendly_name: Cohort Daily Statistics v2
```

Comment:
Are we treating this PR as blocked by https://github.com/mozilla/bigquery-etl/pull/4482/ ? IMO we probably should, and then add a `startup_profile_selection_reason` column from `clients_first_seen_v2`

Reply:
I can add that